packages/jest-leak-detector/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
.npmignore is still missing from this folder. Please copy it from another package.